### PR TITLE
Update for material ui 1.0

### PR DESCRIPTION
**Pull request by tylerthebuildor**

### CODE REVIEW

File: Rating.js [hunk 1]
```diff
@@ -1,88 +1,99 @@
...
+  const st = readOnly ? {} : styles.editable;
+  if (checked) return <Star style={st} color={orange['500']} {...p} />;
+  else if (hovered)
+    return <StarBorder style={st} color={orange['500']} {...p} />;
+  else return <StarBorder style={st} color={grey['500']} {...p} />;
 };
 
 class Rating extends React.Component {
+  static defaultProps = {
+    onChange: () => console.log('hi'),
```
Comment (lawrentiy) [inline Rating.js:25]: It`ll be better to write some debug information for developer or some instructions m.b.
 instead of just "hi" ;)

**Review by lawrentiy — APPROVED** [review]

File: Rating.js (hunk 1 above)
Comment (tylerthebuildor) [reply Rating.js:25]: Yeah haha 😀 forgot I put that in there to be honest. I'll submit another PR to fix it. Also would love to get this package on NPM if I do the leg work of setting up a package.json for you will you publish it?

Comment (lawrentiy) [reply Rating.js:25]: Niiice. Do it and I`ll check, do merge and will publish

Comment (tylerthebuildor) [reply Rating.js:25]: Awesome! 🎉

**Pull request merged**